### DISCUSSION TITLE
Ticket payment error in bricks.

### DISCUSSION THREAD

**Discussion by Pacocc97** [mercadopago/sdk-js#178] [Q&A]
Im getting an error when I try to create a ticket payment. I send the next data.
I don´t know what is the issue, I´m doing the same as the documents.
![257366814-5100a431-39f6-4966-a421-8c46fcb579ab](https://github.com/mercadopago/sdk-js/assets/108708176/75552834-645b-42a8-970c-cf2e05fe94a4)

**Comment by lucmantovani** [comment]
Duplicates https://github.com/mercadopago/sdk-nodejs/discussions/244
